src/main/kotlin/apply/domain/evaluationtarget/EvaluationTarget.kt

```diff
@@ -38,8 +38,14 @@ class EvaluationTarget(
     val isPassed: Boolean
         get() = this.evaluationStatus == EvaluationStatus.PASS
 
-    fun submitted(): Boolean {
-        return evaluationStatus != EvaluationStatus.FAIL || !evaluationAnswers.allZero()
+    fun passIfBeforeEvaluation() {
+        if (evaluationStatus == EvaluationStatus.WAITING) {
+            evaluationStatus = EvaluationStatus.PASS
+        }
+    }
+
+    fun evaluated(): Boolean {
+        return !evaluationAnswers.allZero() || evaluationStatus != EvaluationStatus.FAIL
```

Comment:
```suggestion
        return !(evaluationAnswers.allZero() && evaluationStatus == EvaluationStatus.FAIL)
```

혹시 이 코드와 같은 내용일까요? `부정 or 부정` 조건보다 `!(긍정 and 긍정)`이 이해하기 더 쉬울 것 같기도 한데 의견이 어떤지 궁금하네요!ㅎㅎㅎ

Reply:
음 이 부분도 특별히 제 기호가 없어 팀원들 스타일 듣고 따르겠습니다.

Reply:
저도 배럴이 제시해준 코드가 더 읽기 쉬운 것 같아요 ㅎㅎ

Reply:
처리했습니다.